mkdocs/tests/utils_tests.py

```diff
@@ -67,3 +68,34 @@ def test_create_media_urls(self):
         for path, expected_result in expected_results.items():
             urls = utils.create_media_urls(site_navigation, [path])
             self.assertEqual(urls[0], expected_result)
+
```

Comment:
I don't think the Py26 unittest lib has `skipIf`.  Seems to be new in Py31 https://docs.python.org/3/library/unittest.html#skipping-tests-and-expected-failures.